project/Libs.scala

```diff
@@ -0,0 +1,12 @@
+package nexus.builds
```

Comment:
This probably shouldn't be called `Libs` since it is backend-dependent? Or just put this in the main `lazy val n4dj` part?